### DISCUSSION TITLE
可以帮我配置一下嘛？谢谢

### DISCUSSION THREAD

**Discussion by ZYC-888** [swar/Swar-Chia-Plot-Manager#623] [General]
操作系统	Windows 10 64位 ( DirectX 12 )
	
处理器	AMD Ryzen 9 5900X 12-Core 12核24线程
ssd     2T*2
hhd     希捷 HDD 8T * 10
主板	华硕 TUF GAMING X570-PLUS ( AMD PCI 标准主机 CPU 桥 )
内存	64 GB ( 未知 DDR4 2666MHz )
主硬盘	金士顿 SA400S37120G ( 120 GB / 固态硬盘 )
主显卡	Nvidia GeForce GT 520 ( 512 MB / Nvidia )

**Comment by ckokt** [comment]
https://github.com/swar/Swar-Chia-Plot-Manager/discussions/561

**Reply by ZYC-888** [reply]
谢谢
